metis-media-service/src/main/java/eu/europeana/metis/mediaservice/WebResource.java

```diff
@@ -0,0 +1,160 @@
+package eu.europeana.metis.mediaservice;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import eu.europeana.corelib.definitions.jibx.AudioChannelNumber;
+import eu.europeana.corelib.definitions.jibx.BitRate;
+import eu.europeana.corelib.definitions.jibx.CodecName;
+import eu.europeana.corelib.definitions.jibx.ColorSpaceType;
+import eu.europeana.corelib.definitions.jibx.DoubleType;
+import eu.europeana.corelib.definitions.jibx.Duration;
+import eu.europeana.corelib.definitions.jibx.HasColorSpace;
+import eu.europeana.corelib.definitions.jibx.HasMimeType;
+import eu.europeana.corelib.definitions.jibx.Height;
+import eu.europeana.corelib.definitions.jibx.HexBinaryType;
+import eu.europeana.corelib.definitions.jibx.IntegerType;
+import eu.europeana.corelib.definitions.jibx.LongType;
+import eu.europeana.corelib.definitions.jibx.NonNegativeIntegerType;
+import eu.europeana.corelib.definitions.jibx.OrientationType;
+import eu.europeana.corelib.definitions.jibx.SampleRate;
+import eu.europeana.corelib.definitions.jibx.SampleSize;
+import eu.europeana.corelib.definitions.jibx.SpatialResolution;
+import eu.europeana.corelib.definitions.jibx.StringType;
+import eu.europeana.corelib.definitions.jibx.Type1;
+import eu.europeana.corelib.definitions.jibx.WebResourceType;
+import eu.europeana.corelib.definitions.jibx.Width;
+
+public class WebResource {
+	private final WebResourceType resource;
+	
+	WebResource(WebResourceType resource) {
+		this.resource = resource;
+	}
+	
+	public void setWidth(int width) {
+		resource.setWidth(intVal(new Width(), width));
+	}
+	
+	public void setHeight(int height) {
+		resource.setHeight(intVal(new Height(), height));
+	}
+
+	public void setMimeType(String mimeType) {
+		HasMimeType hasMimeType = new HasMimeType();
+		hasMimeType.setHasMimeType(mimeType);
+		resource.setHasMimeType(hasMimeType);
+	}
+	
+	public void setFileSize(long fileSize) {
+		resource.setFileByteSize(longVal(fileSize));
+	}
+	
+	public void setColorspace(String colorspace) {
+		if (!"grayscale".equals(colorspace) && !"sRGB".equals(colorspace))
+			throw new IllegalArgumentException("Unrecognized color space: " + colorspace);
+		HasColorSpace hasColorSpace = new HasColorSpace();
+		hasColorSpace.setHasColorSpace("sRGB".equals(colorspace) ? ColorSpaceType.S_RGB : ColorSpaceType.GRAYSCALE);
+		resource.setHasColorSpace(hasColorSpace);
+	}
+	
+	public void setOrientation(boolean landscape) {
```

Comment:
Create two methods instead of using the boolean as parameter. It is usually cleaner to the user of the method what the method does.